yeezyCopper.js

```diff
@@ -109,7 +108,7 @@ function checkovernightShip()
     sleep(500);
     console.log('We got Overnight Shipping! Now placing order!');
     saveDelivery();
-  
+
   if (shipfast === false) {
```

Comment:
since you put shipfast in the config file, you'll need to change any instances of `shipfast` to `config.shipfast`. You can use your editor's find and replace tool to make this easier.